verilog/analysis/checkers/suspicious_semicolon_rule.cc

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017-2023 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "suspicious_semicolon_rule.h"
```

Comment:
The include file should be fully qualified from the root of the project:

```suggestion
#include "verilog/analysis/checkers/suspicious_semicolon_rule.h"
```